requirements-linters.txt

```diff
@@ -3,6 +3,7 @@ flake8-broken-line
 flake8-bugbear
 flake8-builtins
 flake8-comprehensions
+flake8-copyright
```

Comment:
Does it validate all header's lines?
```
# Copyright (C) 2021-2022 Intel Corporation
# SPDX-License-Identifier: Apache-2.0
```
Maybe we should chenge default regex?

Reply:
Why? Default regex describe that part: ` Copyright (C) 2021-2022 `. We should check something else?

Reply:
It goesn't check `SPDX-License-Identifier: Apache-2.0`